annotation/compiler/build.gradle

```diff
@@ -1,10 +1,22 @@
 import org.gradle.internal.jvm.Jvm
+import java.nio.file.Files
+import static java.nio.file.StandardCopyOption.*
 
 apply plugin: 'java'
 
+configurations {
+    // adapted from https://android.googlesource.com/platform/frameworks/testing/+/976c423/espresso/espresso-lib/build.gradle
+    // compileOnly dependencies will be repackaged, see rules in jarjar ant task below
+    jarjar 
+}
+
 dependencies {
-    compile 'com.squareup:javapoet:1.9.0'
-    compile 'com.google.auto.service:auto-service:1.0-rc3'
+    // from https://code.google.com/archive/p/jarjar/downloads
+    jarjar files('libs/jarjar-1.4.jar')
+
+    compileOnly 'com.squareup:javapoet:1.9.0'
+    compileOnly 'com.google.auto.service:auto-service:1.0-rc3'
+
     compile 'com.google.code.findbugs:jsr305:3.0.1'
```

Comment:
compileOnly here is fine since these are just annotations anyway. Doing so will resolve the findbugs conflict as well as those with javapoet and auto-service.